examples/sveltekit-openai/package.json

```diff
@@ -1,33 +1,46 @@
 {
   "name": "sveltekit-openai",
-  "version": "0.0.0",
   "private": true,
   "scripts": {
-    "dev": "vite dev --port 3000",
+    "dev": "vite dev",
     "build": "vite build",
     "preview": "vite preview",
-    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
-    "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch"
-  },
-  "dependencies": {
-    "@ai-sdk/openai": "1.2.5",
-    "@ai-sdk/svelte": "1.1.24",
-    "ai": "4.1.61",
-    "openai": "4.52.6",
-    "zod": "^3.23.8"
+    "prepare": "svelte-kit sync || echo ''",
+    "type-check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+    "type-check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+    "prettier-fix": "prettier --write .",
+    "prettier-check": "prettier --check .",
+    "lint": "eslint .",
+    "clean": "rm -rf .svelte-kit ** rm -rf .vercel"
   },
+  "type": "module",
   "devDependencies": {
-    "@fontsource/fira-mono": "^5",
-    "@neoconfetti/svelte": "^2",
-    "@sveltejs/adapter-auto": "^3",
-    "@sveltejs/kit": "^2",
-    "@sveltejs/vite-plugin-svelte": "^3",
-    "@types/cookie": "^1.0.0",
-    "svelte": "^5",
-    "svelte-check": "^4.0",
-    "tslib": "^2.6.0",
-    "typescript": "5.6.3",
-    "vite": "^5"
-  },
-  "type": "module"
+    "@ai-sdk/openai": "workspace:*",
```

Comment:
the reason we were using fixed versions was that each example is a vercel-deployable template (for debugging if needed)

Reply:
Ah oops, I don't think I even did this on purpose -- I'll fix it